### ISSUE TITLE
No Python 3 support

### ISSUE THREAD

**Issue by jfach** [pschmitt/guacapy#15]
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/opt/pycharm-2017.2.4/helpers/pydev/_pydev_bundle/pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
  File "/home/jordan/.virtualenvs/firefly-env-py3/lib/python3.6/site-packages/guacapy/__init__.py", line 1, in <module>
    from client import Guacamole
  File "/opt/pycharm-2017.2.4/helpers/pydev/_pydev_bundle/pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'client'
```

I might be able to come back to this at one point and put in PR to remedy this. It doesn't look to be a huge issue, I'm assuming its just the way the imports are handled.

**Comment by jfach** [comment]
@pschmitt I opened a PR for this issue, #16

**Comment by pschmitt** [comment]
Fixed by #16
